benchmarks/shelley3pools/submit_delegation_tx.sh

```diff
@@ -9,12 +9,12 @@ CLICMD=${CLICMD:-"run cardano-cli"}
 # === submit delegation transactions ===
```

Comment:
is this still in use?

Reply:
This might get back into use, if we choose to fix the pools in Cardano mode -- currently they don't work, because genesis staking is ignored by the system.